src/rust/dbv2/migrations/2023-10-06-164142_leaderboards/up.sql

```diff
@@ -0,0 +1,43 @@
+-- Your SQL goes here
+CREATE TABLE competition_metadata (
+    id SERIAL NOT NULL PRIMARY KEY,
+    start TIMESTAMPTZ NOT NULL,
+    "end" TIMESTAMPTZ NOT NULL,
+    prize INT NOT NULL,
+    market_id NUMERIC NOT NULL,
+    frontends_required TEXT[] NOT NULL
```

Comment:
How about `integrators_required`? 

We are using integrators as a proxy for frontend but the data says integrators so I say we just stick with that